components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js

```diff
@@ -2319,5 +2335,42 @@ define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dr
             return partitionId + '_pc' + (maxId + 1);
         };
 
+        DesignGrid.prototype.multipleSelection = function () {
+            var self = this;
+            var selector = document.createElement('div');
+            selector.setAttribute("id", "selector");
```

Comment:
let's just add a div with specified unique class and add corresponding styles in appropriate .css file

Reply:
Fixed with f0466f5ca9b5358f55e4373e43112bc390ccfd27